contributing/development/handling_compatibility_breakages.rst

```diff
@@ -0,0 +1,141 @@
+.. _doc_handling_compatibility_breakages:
+
+Handling compatibility breakages
+================================
+
+.. TODO: Elaborate on types of compatibility and procedure.
+
+So you've added a new parameter to a method, changed the return type,
+changed the type of a parameter, or changed its default value,
+and now the automated testing is complaining about compatibility breakages?
+
+Breaking compatibility should be avoided, but when necessary there are systems in place
+to handle this in a way that makes the transition as smoothly as possible.
```

Comment:
```suggestion
to handle this in a way that makes the transition as smooth as possible.
```
This was better before, "smooth" is an adjective referring to "transition".

"smoothly" as an adverb would refer to a verb, e.g. "it transitions smoothly" :wink:

Reply:
Yep, realized after, only read part of the sentence when fixing